app/components/audio/song-control.tsx

```
@@ -1,57 +1,154 @@
-            {/* </div> */}
+            <input 
+              className={twMerge(
+                "max-w-28 w-full",
+                isDisabled && "opacity-50"
+              )}
+              aria-label="volume" 
+              type="range" 
+              value={volume * 100} 
+              min={0} 
+              max={100}
+              onChange={(e) => setVolume(parseInt(e.target.value) / 100)}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)